apps/i18n/dance/en_us.json

```diff
@@ -1,22 +1,30 @@
 {
-  "danceFeedbackNeedNewDancer": "You need to make a dancer.",
-  "danceFeedbackNoDancers": "You have no dancers.",
-  "danceFeedbackNoBackground": "You need to add a background effect.",
-  "danceFeedbackTooManyDancers": "Be careful! If you put `make a new` block inside `every 2 measures`, you will create a lot of dancers.",
-  "danceFeedbackUseSetSize": "Use the `set backup_dancer2 size` block to make that dancer smaller.",
-  "danceFeedbackUseSetTint": "Use the `set tint` block to change a dancer's tint.",
-  "danceFeedbackUseStartMapping": "Try adding the <xml><block type=\"Dancelab_startMapping\"><title name=\"SPRITE\">right_unicorn</title><title name=\"PROPERTY\">\"scale\"</title><title name=\"RANGE\">\"bass\"</title></block></xml> block to your program.",
-  "danceFeedbackStartNewMove": "Your dancer wasn't doing a new move after the fourth measure.",
+ "aiDanceFeedbackAiBlockNoParameters": "Choose 3 emojis that match the mood of the song and prompt AI to generate your stage.",
```

Comment:
It seems we should maybe prefix these `danceAi` rather than `aiDance`, to be consistent with the hierarchy used in the other keys?

Reply:
Would you want to update these in https://github.com/code-dot-org/code-dot-org/pull/54619? Once you do, I'll update the validation code on levelbuilder.

Reply:
It sounds like the validation strings are likely to get new revisions as the wording is finalized this week, so maybe we can do it in a PR that also does that?